src/components/sign-button/index.tsx

```diff
@@ -0,0 +1,48 @@
+"use client";
+import { signOut, useSession } from "next-auth/react";
+import Image from "next/image";
+import Link from "next/link";
+import React from "react";
+
+export const SignInButton = () => {
+    const { data: session } = useSession();
+
+    if (session && session.user)
+        return (
+            <div
+                className={
+                    "px-7   border-l-2 gap-4 border-l-heliotrope flex justify-between items-center"
+                }
```

Comment:
If you have a string prop, you can omit `{}`
```suggestion
                className="px-7   border-l-2 gap-4 border-l-heliotrope flex justify-between items-center"
```